examples/elvis-operator/elvis_operator.bal

```diff
@@ -1,16 +1,18 @@
 import ballerina/io;
 
 public function main() {
-    string|() x = null;
+    string|() x = ();
 
-    // If you need to get the string value of x, and if the value of x is `null`, you may want
-    // to assign a known value. This is how you can do it via Match
-    // Expression.
-    string matchExprOutput = x is string ? s : "value is null";
-
-    io:println("The output from match expression: ", matchExprOutput);
+    // If you need to get the `string` value of x, and if the value of x is `nil`, you may want
+    // to assign a known value. This is how you can do it via type-guard (other word `is` check)
```

Comment:
Change to "This is how you can do it via type-guard (which is also known as `is` check)."